submissions/victorAienobe/php/hobbyArrays.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+    $names = ['Pele','Maradona','Messi','Ronaldo','Neymar','Hazard','Debruyne','Aguero'];
+
+    $hobbies = ['movies','music','chatting','video games','swimming','skating','football','basketball'];
+
+    foreach(array_combine($names,$hobbies) as $firstname => $hobby) {
+            echo 'My name is ' .$firstname . ", I love " . $hobby . "<br>";
+        }
```

Comment:
You haven't formatted it properly. Check the closing curly bracket and the code within the foreach. The tab spaces are the issue